src/lib/modules/tests/test.js

```diff
@@ -56,7 +56,7 @@ class Test {
         if (!account.hexBalance) {
           account.hexBalance = BALANCE_10_ETHER_IN_HEX;
         }
-        return {balance: account.hexBalance, secretKey: account.privateKey};
+        return {balance: account.hexBalance, privateKey: account.privateKey};
```

Comment:
@alaibe - do you know why this is secretKey instead of privateKey? https://github.com/embark-framework/embark/commit/24aecc553bcb1a33982e26f06c2566903a551b88#diff-1ba2275b8a1f44dd8a39179506d1e2edR60
If I do:
1. embark run in the test app
2. In another term, embark test —node embark (in the test app)
3. I’m getting the error:
```
Unsupported account configuration: {"balance":"0x152d02c7e14af6800000","secretKey":"<REMOVED>"}
logger.js:102
Try using one of those: { "privateKey": "your-private-key", "privateKeyFile": "path/to/file/containing/key", "mnemonic": "12 word mnemonic” }
```

In the meantime, I've changed it `privateKey` and the error disappears.

Reply:
Works now after rebasing in your change in master.